external/anomalib/tests/helpers/train.py

```diff
@@ -58,8 +50,13 @@ class OTEAnomalyTrainer:
             Defaults to "./ote/configs/template.yaml".
     """
 
-    def __init__(self, model_template_path: str = "./ote/configs/template.yaml"):
-        dataset_path = os.path.join("./datasets/MVTec", "bottle")
+    def __init__(
+        self,
+        model_template_path: str = "./ote/configs/template.yaml",
```

Comment:
I know it doesn't change the overall behaviour of the test, but maybe we could change the default path here as well